Frontend/src/pages/Login/Form/Form.js

```diff
@@ -0,0 +1,131 @@
+import {useForm} from 'react-hook-form';
+import {Form, Label, Input} from 'reactstrap';
+import {withRouter} from 'react-router-dom';
+import PropTypes from 'prop-types';
+import {useDispatch} from 'react-redux';
+import ErrorMessage from '../../../components/ErrorMessage';
+import {emailValidation, passwordValidation} from '../../../constant';
+import '../../../styles/Form.scss';
+import {fetchUserDataSuccess} from '../../../actions/user';
+
+const LoginForm = ({history}) => {
+  const dispatch = useDispatch();
+  const {register, handleSubmit, errors, reset, watch, formState} = useForm({
+    mode: 'onChange',
+  });
+  const email = watch('email');
+  const password = watch('password');
+  const onSubmit = (data) => {
+    const emailid = data.email;
+    localStorage.setItem('sabkaBazar', JSON.stringify({emailid}));
```

Comment:
extract local storage logic to some other file. Form component should not be aware of of any storage mechanism

Reply:
local storage logic moved to user action file.